kong/plugins/rate-limiting/policies/init.lua

```diff
@@ -1,21 +1,65 @@
-local singletons = require "kong.singletons"
-local timestamp = require "kong.tools.timestamp"
-local redis = require "resty.redis"
 local policy_cluster = require "kong.plugins.rate-limiting.policies.cluster"
+local singletons = require "kong.singletons"
 local reports = require "kong.reports"
+local redis = require "resty.redis"
+local luatz = require "luatz"
 
 
-local ngx_log = ngx.log
-local shm = ngx.shared.kong_rate_limiting_counters
+local timetable = luatz.timetable
 local pairs = pairs
+local floor = math.floor
+local null = ngx.null
+local shm = ngx.shared.kong_rate_limiting_counters
 local fmt = string.format
 
 
-local NULL_UUID = "00000000-0000-0000-0000-000000000000"
+local log_err = kong and kong.log.err or function(...)
```

Comment:
Is this here because of tests where `kong` is not available? If that's the case, I'd rather just populate `kong.log.err` in the test case before testing this module instead of leaking `ngx.log` here.